org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/editor/contentassist/AbstractIdeTemplateProposalProvider.java

```diff
@@ -54,10 +54,7 @@ public Variable(final String name) {
     @Override
     @Pure
     public int hashCode() {
-      final int prime = 31;
-      int result = 1;
-      result = prime * result + ((this.name== null) ? 0 : this.name.hashCode());
-      return result;
+      return 31 * 1 + ((this.name== null) ? 0 : this.name.hashCode());
```

Comment:
That could be slightly nicer but not a dealbreaker.

Reply:
well the change in lib is already comitted. this is just the as is